### ISSUE TITLE
Token function can't use color opacity modifier

### ISSUE THREAD

**Issue by jungai** [chakra-ui/panda#2367]
### Description

When in import token from '@/styled-system/tokens' and use like token('colors.primary.500/20') it cause type error `Argument of type '"colors.primary.500/20"' is not assignable to parameter of type 'Token'`

### Link to Reproduction

https://play.panda-css.com/OsKzQNo8eC

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### JS Framework

Nextjs(App router)

### Panda CSS Version

0.34.3

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by astahmer** [comment]
yeah this one is not an issue, we support directly transforming the string token fn like [`border: "1px solid token(colors.amber.300/50)`](https://play.panda-css.com/OsKzQNo8eC/gzubagys8G) but not in the JS `token` fn (the one that you have to import from `styled-system`)

you can achieve the same result with string concatenation on your side with something like 

```ts
const mix = (value: string, percent: number) => `color-mix(in srgb, ${value} ${percent}%, transparent)`
const tokenValue = token.var('colors.amber.300');
const value = mix(tokenValue)
```

I think it might be worth adding to the documentation somewhere, do you want to take a shot a it ? @jungai
